tests/unit/test_data_validation.py

```diff
@@ -363,6 +366,49 @@
     consts.CONFIG_FILTER_STATUS: None,
 }
 
+# Row validation where we only care about failures and we write them to BQ
+SAMPLE_ROW_CONFIG_BQ_FAILURES = {
+    # BigQuery Specific Connection Config
+    "source_conn": SOURCE_CONN_CONFIG,
+    "target_conn": TARGET_CONN_CONFIG,
+    # Validation Type
+    consts.CONFIG_TYPE: consts.ROW_VALIDATION,
+    # Configuration Required Depending on Validator Type
+    "schema_name": None,
+    "table_name": "my_table",
+    "target_schema_name": None,
+    "target_table_name": "my_table",
+    consts.CONFIG_PRIMARY_KEYS: [
+        {
+            consts.CONFIG_FIELD_ALIAS: "id",
+            consts.CONFIG_SOURCE_COLUMN: "id",
+            consts.CONFIG_TARGET_COLUMN: "id",
+            consts.CONFIG_CAST: None,
+        },
+    ],
+    consts.CONFIG_COMPARISON_FIELDS: [
+        {
+            consts.CONFIG_FIELD_ALIAS: "int_value",
+            consts.CONFIG_SOURCE_COLUMN: "int_value",
+            consts.CONFIG_TARGET_COLUMN: "int_value",
+            consts.CONFIG_CAST: None,
+        },
+        {
+            consts.CONFIG_FIELD_ALIAS: "text_value",
+            consts.CONFIG_SOURCE_COLUMN: "text_value",
+            consts.CONFIG_TARGET_COLUMN: "text_value",
+            consts.CONFIG_CAST: None,
+        },
+    ],
+    consts.CONFIG_RESULT_HANDLER: {
+            consts.CONFIG_TYPE: "BigQuery",
```

Comment:
We have some over-indentation here. We check formatting with `black` as part of running tests. Can you run the command below and commit the changes please:

```
python3 -m black tests/unit/test_data_validation.py
```

Reply:
Thanks @nj1973 - I'll sort this out 👍

Reply:
Cleaned up here -> https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1263/commits/2266a47c6a4ae3be9b09eba05d731afc3d033024